### ISSUE TITLE
Can't respawn

### ISSUE THREAD

**Issue by xnooztvFR** [ErikMast3r/Exclusiv-Altis-Life-new#67]
Hello,

We can not respawn, when I die I do not have the respawn button.

**Comment by xxvirusxx6767** [comment]
And, where is the EAL mods please? :)

**Comment by ErikMast3r** [comment]
Everything is still WIP.... The EAL mods aren't public yet.

And for your problem: wait for the next release. It will be maybe next week.

**Comment by ErikMast3r** [comment]
The problem is the actual EAL mods are public but doesn't work with the latest release. therefore you have to wait for the next release.

**Comment by xxvirusxx6767** [comment]
When will it be available (approximately) ?
I understand, thanks, i wait a update for my problem.

**Comment by ErikMast3r** [comment]
Under projects you can see it. This are the things i have to do before next release.
The EAL mods can be downloaded over Arma 3 Sync. For a tutorial you can click here.
This is the autoconfig link: http://Mods.exclusivgaming.de/mods/lifemodsnew/.a3s/autoconfig

ATTENTION: A big part of this mods aren't my work. If you wanna use it, you have to ask the autor. My own mods are just the "EAL_*" pbo-files. 

ATTENTION 2: This repository won't work with the latest release!

**Comment by xnooztvFR** [comment]
Thanks dude.
If I understand correctly, mods "EAL_*" do not work on the latest version?
Sorry, i am not english ^^

**Comment by ErikMast3r** [comment]
No, in the shops of the latest release are vehicles that aren't in the modset anymore. This was just an example.

**Comment by xnooztvFR** [comment]
Fine. I wait the update, thanks :)

**Comment by ErikMast3r** [comment]
fixed in 1b0e8ae
